### PR TITLE
JS fundamental Assignment

### PR DESCRIPTION
**Pull request by shubhmore1212**
Please review module-1 solution

### CODE REVIEW

File: Course1/Question1/question1_1.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <title>JS Assignment-1</title>
+  </head>
+  <body>
+	<div>
+	  <h1>JS using internal script tag</h1>
+	  <button onclick="testHandler()">Click Me</button>
+	  <p id="demo"></p>
+	</div>
+	<script>
+	     const para=document.getElementById("demo");
```
Comment (vaibhavchalse99) [inline Course1/Question1/question1_1.html]: move this inside the testHandler function-
it should be document.getElementById("demo").innerHTML = `Your name ${name}`
don't break it.

File: Course1/Question1/question1_2.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const para=document.getElementById("demo");
```
Comment (vaibhavchalse99) [inline Course1/Question1/question1_2.js]: move it inside the function testHandler()

File: Course1/Question1/question1_1.html (hunk 1 above)
Comment (shubhmore1212) [reply Course1/Question1/question1_1.html]: Resolved

File: Course1/Question1/question1_2.js (hunk 2 above)
Comment (shubhmore1212) [reply Course1/Question1/question1_2.js]: Resolved

**Review by vaibhavchalse99 — APPROVED** [review]
looks fine

**Pull request merged**